test/acceptance/behave/components/ngsiv2/entities/list_entities/list_entities.feature

```diff
@@ -0,0 +1,808 @@
+  As a context broker user
+  I would like to list all entities with get request and queries parameter using NGSI v2
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+  @happy_path
+  Scenario:  list all entities using NGSI v2 (3 entity groups with 11 entities in total)
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_happy_path  |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    And initialize entity groups recorder
+    And properties to entities
+      | parameter         | value       |
+      | entities_type     | home        |
+      | entities_id       | room1       |
+      | attributes_number | 2           |
+      | attributes_name   | temperature |
+      | attributes_value  | 34          |
+      | attributes_type   | celsius     |
+      | metadatas_number  | 2           |
+      | metadatas_name    | very_hot    |
+      | metadatas_type    | alarm       |
+      | metadatas_value   | random=10   |
+    And create entity group with "5" entities in "normalized" mode
+      | entity | prefix |
+      | id     | true   |
+    And verify that receive several "Created" http code
+    And record entity group
+    And properties to entities
+      | parameter         | value                   |
+      | entities_type     | house                   |
+      | entities_id       | room2                   |
+      | attributes_number | 2                       |
+      | attributes_name   | timestamp               |
+      | attributes_value  | 017-06-17T07:21:24.238Z |
+      | attributes_type   | date                    |
+      | metadatas_number  | 2                       |
+      | metadatas_name    | very_hot                |
+      | metadatas_type    | alarm                   |
+      | metadatas_value   | random=10               |
+    And create entity group with "5" entities in "normalized" mode
+      | entity | prefix |
+      | id     | true   |
+    And verify that receive several "Created" http code
+    And record entity group
+    And properties to entities
+      | parameter        | value      |
+      | entities_type    | "car"      |
+      | entities_id      | "vehicle"  |
+      | attributes_name  | "speed"    |
+      | attributes_value | 89         |
+      | attributes_type  | "km_h"     |
+      | metadatas_name   | "very_hot" |
+      | metadatas_type   | "alarm"    |
+      | metadatas_value  | "hot"      |
+    And create an entity in raw and "normalized" modes
```

Comment:
#### normalized vs keyValues in creation of attributes format

these parameters: `normalized` and `keyValues` are used to define of attributes format in step used to create entities, independently of `options` query parameter. 
Ex:
```
normalized
      {...
           "attr": {
                 "value": "34",
                 "type": "nothing",
                 "metadata": {....}
           }
       }
```
```
keyValues, this parameter ignore of attribute type and metadatas
  {
      "attr": "34"
  }
```

#### raw mode vs not raw

The flow to create an entity or N entities is the following:
   1- `a definition of headers`
   2- `properties to entities`
   3.1- `create entity group with "5" entities in "normalized" mode`: this step create 5 entities, but all fields and valueas are `STRING` with a given prefix and with attributes in normalized mode.
   3.2- `create an entity in raw and "normalized" modes`: create an entity, but the 
fields and values are used as they are literally defined in `a definition of headers` step. `Ex: true, json object, numerics, nul, etc`.
   4.1- verify that receive several "Created" http code
   4.2- verify that receive an "Created" http code

The 3.1 y 3.2 steps are represented in two methods in cb_ngsi_v2.py library, the first uses a `dict` and after the full request is convert to String, the second is create in String directly.